docs/getting-started/quick-start.md

```diff
@@ -101,6 +101,10 @@ openff-bespoke executor run --file                 "acetaminophen.sdf" \
                             --default-qc-spec xtb gfn2xtb none
 ```
 
+:::{note}
+For XTB (only), `--qc-compute-n-cores` is ignored because of miscommunications between QCEngine and XTB, but this can be worked around by setting the environment variable `OMP_NUM_THREADS`. See the FAQ for details.
+:::
```

Comment:
A direct link would be nice but I chose not to dedicate the time to figuring that out. It should remain a single click away in the rendered docs